monitoring/rid_qualifier/test_create_flight_record_from_kml.py

```diff
@@ -0,0 +1,21 @@
+from monitoring.rid_qualifier import create_flight_record_from_kml as frk
+
+def test_get_interpolated_altitude(mocker):
```

Comment:
Having tests is great -- let's write a brief comment in this file (or perhaps in this folder's readme) saying how someone should invoke these tests

Reply:
Let's still write this comment, either in this file (probably preferable) or in the README, even though I think it is as simple as running `pytest` from this working directory.  However, also note that the tests fail currently, I think because the function was renamed from `get_interpolated_altitude` to `get_interpolated_value`.